### ISSUE TITLE
GFS soundings in v1.3.0 Xenia

### ISSUE THREAD

**Issue by TonyLyza** [sharppy/SHARPpy#132]
When loading GFS soundings in v1.3.0 Xenia, there's an issue with the time step that leads to a limitation in the time range of output available.  The gui allows for selection of soundings every 3 hours from F000 to F180.  However, when these soundings are loaded, soundings for every hour from F000 to F060 are now given.  I assume this is an issue with a change in what the distribution source provides but wanted to bring it to your guys' attention.  We've tried this both on several machines, both with Xenia installed from source on Linux and in the pre-compiled binary in Windows 10.  FWIW, it does appear that the time steps listed at the upper-left on the soundings are the actual time steps given (i.e. it's not the gui mislabelling the time steps) as the soundings shown are what I would expect for the time steps shown.

**Comment by tsupinie** [comment]
I'm pretty sure the issue is that on the GFS upgrade back in July, the Bufkit output became available every hour out to 120 hours and every three hours thereafter out to 180. However, SHARPpy was still assuming the old three-hourly out to 180 hours, which meant the picker would display the wrong times.

I've updated the code in the master branch to at least do the hourly out to 120 hours. I didn't plan for multiple time intervals over a single run when I wrote the configuration code, so I'll have to give some thought as to how I'll go out to 180 hours. Also, the update only fixes the source code, not the binary. I'll have to re-compile those at some point.

Let me know if that works.
